README.md

```diff
@@ -19,3 +19,13 @@ variable to the root of your rls checkout:
 ```
 export RLS_ROOT=/Source/rls
 ```  
+## Manually add as regular VSCode extension
+
+If you'd like to test on multiple projects and already have the extension working properly, want to load the extension by default.
+
+After following the above instructions, and successfully building the extension once, copy the entire `rls_vscode` directory to either:
```

Comment:
Do I need  to restart VSCode after doing this or does it just detect the change?